src/ResourceManager/Network/Commands.Network/NetworkWatcher/TestAzureNetworkWatcherConnectivityCommand.cs

```diff
@@ -113,6 +113,16 @@ public override void Execute()
 
             MNM.ConnectivityParameters parameters = new MNM.ConnectivityParameters();
 
+            if (string.IsNullOrEmpty(this.DestinationId) && string.IsNullOrEmpty(this.DestinationAddress))
+            {
+                throw new ArgumentException("Connectivity check missing destination resource id or address");
```

Comment:
Please put these customer facing strings into the resource strings for this project